### Titanic competition with TensorFlow Decision Forests
Task: to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition.

Steps involved:
- basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
- how to train a Gradient Boosted Trees (GBT) with default parameters
- how to train a GBT with improved default parameters
- how to tune the parameters of a GBTs
- how to train and ensemble many GBTs

In [3]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.4.0


In [4]:
# import dataset
train_df = pd.read_csv("train.csv")
serving_df = pd.read_csv("test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

- Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
- Extract any prefix in the ticket. For example: ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [5]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

In [7]:
# remove the input features: ticket, passenger_id, survived, ticket_number
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


In [8]:
# convert pandas dataset to tensorflow

def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

Metal device set to: Apple M1


In [9]:
# train a GradientBoostedTreesModel model with the default parameters
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 23-07-13 19:34:11.5229 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 19:34:11.5255 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 19:34:11.5255 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
2023-07-13 19:34:11.675965: W tensorflow/tsl/platform/profile_utils/cpu_utils.cc:128] Failed to get CPU frequency: 0 Hz
[INFO 23-07-13 19:34:13.9141 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpetehxzjx/model/ with prefix ce352b1978de4bb1
[INFO 23-07-13 19:34:13.9168 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-07-13 19:34:13.9168 EDT kernel.cc:1075] Use fast generic engine


Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: could not get source code
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: could not get source code
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
Accuracy: 0.804347813129425 Loss:0.8922085762023926


In [10]:
# Train model with improved default parameters

model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 23-07-13 19:34:44.5052 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 19:34:44.5053 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 19:34:44.5053 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.77173912525177 Loss:1.0344746112823486


[INFO 23-07-13 19:34:44.8409 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpr31bu0gb/model/ with prefix 6146817755ee4696
[INFO 23-07-13 19:34:44.8437 EDT decision_forest.cc:660] Model loaded with 34 root(s), 1782 node(s), and 10 input feature(s).
[INFO 23-07-13 19:34:44.8437 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesGeneric" built
[INFO 23-07-13 19:34:44.8437 EDT kernel.cc:1075] Use fast generic engine


In [11]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1
Trainable params: 0
Non-trainable params: 1
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.542176 ################
    2.           "Age"  0.362914 ########
    3.          "Fare"  0.267978 ####
    4.          "Name"  0.210150 #
    5.        "Pclass"  0.182221 
    6. "Ticket_number"  0.182105 
    7.      "Embarked"  0.177187 
    8.   "Ticket_item"  0.176830 
    9.         "Parch"  0.176170 
   10.         "SibSp"  0.171814 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 28.000000 ################
    2. "Name"  6.000000

In [16]:
# make predictions

def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="https://www.kaggle.com/competitions/titanic/submissions" 
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)

Submission exported to https://www.kaggle.com/competitions/titanic/submissions


In [17]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpxrkt5da4 as temporary training directory


[WARNING 23-07-13 21:55:29.7670 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:55:29.7671 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:55:29.7671 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.8767123222351074 Loss:0.6833678483963013


[INFO 23-07-13 21:55:59.5575 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpxrkt5da4/model/ with prefix 75f96d65ad7f48cc
[INFO 23-07-13 21:55:59.5630 EDT decision_forest.cc:660] Model loaded with 35 root(s), 1085 node(s), and 12 input feature(s).
[INFO 23-07-13 21:55:59.5630 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesGeneric" built
[INFO 23-07-13 21:55:59.5630 EDT kernel.cc:1075] Use fast generic engine


In [18]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

[WARNING 23-07-13 21:55:59.7124 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:55:59.7125 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:55:59.7125 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:0


[INFO 23-07-13 21:55:59.9213 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp00vx_f4d/model/ with prefix 91f19135562a4028
[INFO 23-07-13 21:55:59.9231 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:00.0986 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:00.0986 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:00.0986 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:1


[INFO 23-07-13 21:56:00.3672 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpy5x2g8e7/model/ with prefix 6c0e8f79b7334e30
[INFO 23-07-13 21:56:00.3711 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:00.5379 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:00.5380 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:00.5380 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:2


[INFO 23-07-13 21:56:00.8145 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpp99fanif/model/ with prefix cb2eed5d92de439c
[INFO 23-07-13 21:56:00.8185 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:00.9875 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:00.9875 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:00.9876 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:3


[INFO 23-07-13 21:56:01.2310 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp41yb4w18/model/ with prefix b852548c6e814768
[INFO 23-07-13 21:56:01.2339 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:01.4051 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:01.4051 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:01.4051 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:4


[INFO 23-07-13 21:56:01.6130 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpetrv8vja/model/ with prefix 19cbdaa545ec4b6e
[INFO 23-07-13 21:56:01.6147 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:01.7851 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:01.7851 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:01.7851 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:5


[INFO 23-07-13 21:56:02.0003 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpy496k1hp/model/ with prefix e788e7e3c38f4d3e
[INFO 23-07-13 21:56:02.0023 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:02.1750 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:02.1751 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:02.1751 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:6


[INFO 23-07-13 21:56:02.4306 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpch1e3b_w/model/ with prefix 2a9bcb5dedbc4492
[INFO 23-07-13 21:56:02.4340 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:02.6035 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:02.6035 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:02.6035 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:7


[INFO 23-07-13 21:56:02.8293 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpdtiiyqq5/model/ with prefix d5bd1f6d5c7a474a
[INFO 23-07-13 21:56:02.8316 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:02.9998 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:02.9998 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:02.9998 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:8


[INFO 23-07-13 21:56:03.2862 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpo7g3u0or/model/ with prefix c3ce8c87e3734032
[INFO 23-07-13 21:56:03.2906 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:03.4603 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:03.4603 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:03.4603 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:9


[INFO 23-07-13 21:56:04.1975 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpzbbclur3/model/ with prefix a95218ed8b9e4acf
[INFO 23-07-13 21:56:04.2032 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:04.3775 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:04.3776 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:04.3776 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:10


[INFO 23-07-13 21:56:04.6097 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp4gt3cpgc/model/ with prefix 2128bbe41669437e
[INFO 23-07-13 21:56:04.6123 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:04.7811 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:04.7812 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:04.7812 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:11


[INFO 23-07-13 21:56:05.0869 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpzqfa2_nb/model/ with prefix 231820c569794ca0
[INFO 23-07-13 21:56:05.0921 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:05.2605 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:05.2605 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:05.2605 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:12


[INFO 23-07-13 21:56:05.5145 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpqdngh36n/model/ with prefix 84cf0dfb3a64440e
[INFO 23-07-13 21:56:05.5178 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:05.6895 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:05.6895 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:05.6895 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:13


[INFO 23-07-13 21:56:06.1348 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpld_3tbmg/model/ with prefix 724cd96040604e80
[INFO 23-07-13 21:56:06.1447 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:06.3182 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:06.3183 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:06.3183 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:14


[INFO 23-07-13 21:56:06.5521 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp710xtk90/model/ with prefix 0badb10c1e87466d
[INFO 23-07-13 21:56:06.5547 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:06.7242 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:06.7242 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:06.7242 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:15


[INFO 23-07-13 21:56:06.9397 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpv43h7w2g/model/ with prefix f9d7d5ff8e5740ad
[INFO 23-07-13 21:56:06.9415 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:07.1117 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:07.1117 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:07.1117 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:16


[INFO 23-07-13 21:56:07.3782 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp0b2ji9xm/model/ with prefix 25f34659ff95460d
[INFO 23-07-13 21:56:07.3820 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:07.5508 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:07.5509 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:07.5509 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:17


[INFO 23-07-13 21:56:07.7778 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp2z_lvedt/model/ with prefix d08ef77a4f9a4f0b
[INFO 23-07-13 21:56:07.7801 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:07.9493 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:07.9494 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:07.9494 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:18


[INFO 23-07-13 21:56:08.2375 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpy3xjcanh/model/ with prefix e30b81b69ba24419
[INFO 23-07-13 21:56:08.2419 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:08.4134 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:08.4134 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:08.4134 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:19


[INFO 23-07-13 21:56:08.7114 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpioi05xb9/model/ with prefix e8ef8aed0ee84112
[INFO 23-07-13 21:56:08.7159 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:08.8858 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:08.8858 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:08.8858 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:20


[INFO 23-07-13 21:56:09.1310 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp9semcnc1/model/ with prefix d9375eb1880f499e
[INFO 23-07-13 21:56:09.1332 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:09.3053 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:09.3053 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:09.3053 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:21


[INFO 23-07-13 21:56:09.5330 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp1dvt10g7/model/ with prefix 1be82351edc0443e
[INFO 23-07-13 21:56:09.5353 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:09.7060 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:09.7060 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:09.7060 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:22


[INFO 23-07-13 21:56:09.9642 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmprfc97z_4/model/ with prefix 82207e33684f445f
[INFO 23-07-13 21:56:09.9674 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-07-13 21:56:09.9675 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:10.1380 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:10.1381 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:10.1381 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:23


[INFO 23-07-13 21:56:10.3667 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpzvbrdmlk/model/ with prefix 0244f93ffa024885
[INFO 23-07-13 21:56:10.3690 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:10.5476 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:10.5476 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:10.5477 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:24


[INFO 23-07-13 21:56:10.7752 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp6597wlik/model/ with prefix 79ecfaef4b0d4ce8
[INFO 23-07-13 21:56:10.7776 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:10.9558 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:10.9559 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:10.9559 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:25


[INFO 23-07-13 21:56:11.2089 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpiwzkaiwq/model/ with prefix 0e66611345fe47fc
[INFO 23-07-13 21:56:11.2120 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:11.3835 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:11.3835 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:11.3835 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:26


[INFO 23-07-13 21:56:11.6009 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpd98s2n4n/model/ with prefix ffee0a79e6984d27
[INFO 23-07-13 21:56:11.6028 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:11.7832 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:11.7833 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:11.7833 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:27


[INFO 23-07-13 21:56:12.0657 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpqajmi236/model/ with prefix 2145bcff07c94706
[INFO 23-07-13 21:56:12.0701 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:12.2461 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:12.2461 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:12.2461 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:28


[INFO 23-07-13 21:56:12.4463 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp2bz9xn5h/model/ with prefix 6c102ff772c949f5
[INFO 23-07-13 21:56:12.4525 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:12.6232 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:12.6232 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:12.6232 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:29


[INFO 23-07-13 21:56:12.8427 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp9cdpqx_q/model/ with prefix 1c43c12f4c2d4d83
[INFO 23-07-13 21:56:12.8446 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:13.0161 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:13.0161 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:13.0161 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:30


[INFO 23-07-13 21:56:13.2901 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp5v9wrjme/model/ with prefix 4b4c23a60ae44457
[INFO 23-07-13 21:56:13.2940 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:13.4645 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:13.4646 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:13.4646 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:31


[INFO 23-07-13 21:56:13.8087 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpmdf22rqv/model/ with prefix 88aea7cc6d854040
[INFO 23-07-13 21:56:13.8144 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:13.9860 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:13.9860 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:13.9860 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:32


[INFO 23-07-13 21:56:14.2214 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmppemqjo8d/model/ with prefix 12eb60b94275496d
[INFO 23-07-13 21:56:14.2239 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:14.3931 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:14.3931 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:14.3931 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:33


[INFO 23-07-13 21:56:14.6436 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp5oykyddc/model/ with prefix 16bdb1c77878484e
[INFO 23-07-13 21:56:14.6467 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:14.8188 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:14.8189 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:14.8189 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:34


[INFO 23-07-13 21:56:15.0464 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmps0_1dffg/model/ with prefix 327c96779ed74836
[INFO 23-07-13 21:56:15.0487 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:15.2189 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:15.2189 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:15.2189 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:35


[INFO 23-07-13 21:56:15.4772 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpofsc05oo/model/ with prefix 601c075d0db84668
[INFO 23-07-13 21:56:15.4813 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:15.6558 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:15.6558 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:15.6559 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:36


[INFO 23-07-13 21:56:15.8794 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpb1ad0ss4/model/ with prefix 166804b9519a4528
[INFO 23-07-13 21:56:15.8815 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:16.0556 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:16.0557 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:16.0557 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:37


[INFO 23-07-13 21:56:16.2752 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpg1hkh5uy/model/ with prefix 71b60b735e604c9b
[INFO 23-07-13 21:56:16.2776 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:16.4525 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:16.4525 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:16.4525 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:38


[INFO 23-07-13 21:56:16.7172 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpnl2rb96u/model/ with prefix 2ebb7f5f705743af
[INFO 23-07-13 21:56:16.7206 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:16.8952 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:16.8953 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:16.8953 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:39


[INFO 23-07-13 21:56:17.1752 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpq_vuikft/model/ with prefix 003af1084ea14909
[INFO 23-07-13 21:56:17.1792 EDT kernel.cc:1075] Use fast generic engine


i:40


[WARNING 23-07-13 21:56:17.7550 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:17.7551 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:17.7551 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-07-13 21:56:17.9901 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmps4dcsd42/model/ with prefix 4781c9a2909f4896
[INFO 23-07-13 21:56:17.9926 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:18.1682 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:18.1682 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:18.1682 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boostin

i:41


[INFO 23-07-13 21:56:18.4435 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpypgpikar/model/ with prefix 11a856053ec24a48
[INFO 23-07-13 21:56:18.4475 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:18.6184 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:18.6184 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:18.6184 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:42


[INFO 23-07-13 21:56:18.8669 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpbpwu59_y/model/ with prefix 6cad5d46dc5141a3
[INFO 23-07-13 21:56:18.8696 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:19.0466 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:19.0466 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:19.0466 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:43


[INFO 23-07-13 21:56:19.3244 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpg8vq3y4w/model/ with prefix 5378eaf2cf914112
[INFO 23-07-13 21:56:19.3279 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:19.5016 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:19.5017 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:19.5017 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:44


[INFO 23-07-13 21:56:19.7400 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpk8qp2_pg/model/ with prefix 0beb9a6b900248e6
[INFO 23-07-13 21:56:19.7425 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:19.9136 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:19.9136 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:19.9136 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:45


[INFO 23-07-13 21:56:20.1162 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpedctp3ev/model/ with prefix 5c4b68951ea347b8
[INFO 23-07-13 21:56:20.1175 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-07-13 21:56:20.1175 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:20.2937 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:20.2938 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:20.2938 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:46


[INFO 23-07-13 21:56:20.5580 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpfzp9aiy6/model/ with prefix 2fb38ced8a9b4f30
[INFO 23-07-13 21:56:20.5616 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:20.7332 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:20.7332 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:20.7332 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:47


[INFO 23-07-13 21:56:20.9668 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpcm69gycb/model/ with prefix 8e6d9813fc4e4388
[INFO 23-07-13 21:56:20.9692 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:21.1415 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:21.1416 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:21.1416 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:48


[INFO 23-07-13 21:56:21.3638 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp33sgjfiv/model/ with prefix 699e31ef06694051
[INFO 23-07-13 21:56:21.3658 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:21.5389 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:21.5389 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:21.5389 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:49


[INFO 23-07-13 21:56:21.7934 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpz7ccka1n/model/ with prefix 4997f561b52c46eb
[INFO 23-07-13 21:56:21.7965 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:21.9697 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:21.9697 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:21.9697 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:50


[INFO 23-07-13 21:56:22.2709 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmphj7rkbu6/model/ with prefix 1b69ca33cba04457
[INFO 23-07-13 21:56:22.2755 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:22.4512 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:22.4513 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:22.4513 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:51


[INFO 23-07-13 21:56:22.7656 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpdq7gzu5k/model/ with prefix 9d95cce6aeff4aed
[INFO 23-07-13 21:56:22.7707 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:22.9440 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:22.9440 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:22.9440 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:52


[INFO 23-07-13 21:56:23.1901 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpoq9k7qbi/model/ with prefix e029bb26bd9b4d32
[INFO 23-07-13 21:56:23.1929 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:23.3646 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:23.3646 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:23.3646 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:53


[INFO 23-07-13 21:56:23.6028 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpat31dcg9/model/ with prefix 23dd8a1e8674410d
[INFO 23-07-13 21:56:23.6053 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:23.7773 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:23.7773 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:23.7774 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:54


[INFO 23-07-13 21:56:24.0215 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpb3ey8yiz/model/ with prefix 6dda606da42d40ce
[INFO 23-07-13 21:56:24.0243 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:24.1967 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:24.1967 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:24.1967 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:55


[INFO 23-07-13 21:56:24.4651 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp0pei_5ct/model/ with prefix 32c5ec8fa016443d
[INFO 23-07-13 21:56:24.4688 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:24.6417 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:24.6417 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:24.6418 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:56


[INFO 23-07-13 21:56:24.8837 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp1ryo2dsf/model/ with prefix d3f167393c9d4838
[INFO 23-07-13 21:56:24.8862 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:25.0579 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:25.0579 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:25.0579 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:57


[INFO 23-07-13 21:56:25.2676 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp7p8amvl_/model/ with prefix 1219be45b629404f
[INFO 23-07-13 21:56:25.2692 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:25.4419 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:25.4419 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:25.4419 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:58


[INFO 23-07-13 21:56:25.7761 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpvbvjr_ew/model/ with prefix 49e8b6f2b70b4003
[INFO 23-07-13 21:56:25.7818 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:25.9553 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:25.9553 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:25.9553 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:59


[INFO 23-07-13 21:56:26.2778 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpu5l2staj/model/ with prefix 5fc5c13985514b79
[INFO 23-07-13 21:56:26.2828 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:26.4540 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:26.4540 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:26.4540 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:60


[INFO 23-07-13 21:56:26.6684 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpnd068sq0/model/ with prefix 03068237d245470a
[INFO 23-07-13 21:56:26.6700 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:26.8459 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:26.8459 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:26.8459 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:61


[INFO 23-07-13 21:56:27.0853 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpldip8m_h/model/ with prefix aadd7cb8ab764125
[INFO 23-07-13 21:56:27.0879 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:27.2599 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:27.2599 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:27.2599 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:62


[INFO 23-07-13 21:56:27.5395 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpoipaakju/model/ with prefix 34910b83ce7543bc
[INFO 23-07-13 21:56:27.5431 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:27.7373 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:27.7374 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:27.7374 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:63


[INFO 23-07-13 21:56:27.9629 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmple3mpgdm/model/ with prefix a80485121dd44d0b
[INFO 23-07-13 21:56:27.9646 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:28.1517 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:28.1517 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:28.1517 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:64


[INFO 23-07-13 21:56:28.4062 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpwwk23eg5/model/ with prefix 157c6e8b7cbf4687
[INFO 23-07-13 21:56:28.4097 EDT kernel.cc:1075] Use fast generic engine


i:65


[WARNING 23-07-13 21:56:28.6637 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:28.6638 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:28.6638 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-07-13 21:56:28.9731 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp15ls1ti0/model/ with prefix 5f09b181584542de
[INFO 23-07-13 21:56:28.9768 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:29.1535 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:29.1535 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:29.1535 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boostin

i:66


[INFO 23-07-13 21:56:29.3754 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpayjlsy00/model/ with prefix b3365e6ac6984f6e
[INFO 23-07-13 21:56:29.3774 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:29.5473 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:29.5474 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:29.5474 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:67


[INFO 23-07-13 21:56:29.8028 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpnjckk0qf/model/ with prefix df3fa4d5026743e5
[INFO 23-07-13 21:56:29.8058 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:29.9951 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:29.9951 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:29.9951 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:68


[INFO 23-07-13 21:56:30.3475 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpned3m_fl/model/ with prefix 0ca5dfd3f1184d90
[INFO 23-07-13 21:56:30.3535 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-07-13 21:56:30.3535 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:30.5258 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:30.5258 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:30.5259 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:69


[INFO 23-07-13 21:56:30.8027 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpc2smpw9u/model/ with prefix 0359f4727f064f4e
[INFO 23-07-13 21:56:30.8066 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:30.9786 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:30.9786 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:30.9786 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:70


[INFO 23-07-13 21:56:31.2072 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpu7ryi3kf/model/ with prefix e0f3c79221594c90
[INFO 23-07-13 21:56:31.2095 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:31.3792 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:31.3792 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:31.3792 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:71


[INFO 23-07-13 21:56:31.5963 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp__edyt5o/model/ with prefix ee7f70c55e91482c
[INFO 23-07-13 21:56:31.5980 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:31.7665 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:31.7665 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:31.7665 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:72


[INFO 23-07-13 21:56:32.0134 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpdx2gxx7r/model/ with prefix 2c2c5f672a3e4675
[INFO 23-07-13 21:56:32.0162 EDT kernel.cc:1075] Use fast generic engine


i:73


[WARNING 23-07-13 21:56:32.6702 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:32.6702 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:32.6702 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-07-13 21:56:32.8798 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpbxh1yz1b/model/ with prefix 93ecd8cf2c474f0c
[INFO 23-07-13 21:56:32.8813 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:33.0555 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:33.0555 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:33.0555 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boostin

i:74


[INFO 23-07-13 21:56:33.4434 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp3rmvtt8s/model/ with prefix 468de4197afe4fee
[INFO 23-07-13 21:56:33.4510 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:33.6270 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:33.6270 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:33.6270 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:75


[INFO 23-07-13 21:56:34.1291 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpqzdypgpx/model/ with prefix a01310e163304a99
[INFO 23-07-13 21:56:34.1403 EDT kernel.cc:1075] Use fast generic engine


i:76


[WARNING 23-07-13 21:56:34.3377 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:34.3377 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:34.3377 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-07-13 21:56:34.6046 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp3h1w6h9c/model/ with prefix 7070033852cc4265
[INFO 23-07-13 21:56:34.6066 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:34.7885 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:34.7885 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:34.7885 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boostin

i:77


[INFO 23-07-13 21:56:35.0437 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpqpx2lg3l/model/ with prefix 0593c40ece4b493b
[INFO 23-07-13 21:56:35.0467 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:35.2193 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:35.2193 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:35.2193 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:78


[INFO 23-07-13 21:56:35.4290 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpm91uhpz3/model/ with prefix c77ea2d0207f418a
[INFO 23-07-13 21:56:35.4304 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:35.6007 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:35.6008 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:35.6008 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:79


[INFO 23-07-13 21:56:35.8794 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpgib13yuh/model/ with prefix 7f773cc8af4647b2
[INFO 23-07-13 21:56:35.8832 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:36.0702 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:36.0702 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:36.0702 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:80


[INFO 23-07-13 21:56:36.3080 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpk5ea_xnv/model/ with prefix eee0b728c6ce4a69
[INFO 23-07-13 21:56:36.3102 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:36.4891 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:36.4892 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:36.4892 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:81


[INFO 23-07-13 21:56:36.7544 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpctt_e59f/model/ with prefix cfc5d88d711d41bf
[INFO 23-07-13 21:56:36.7578 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:36.9353 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:36.9353 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:36.9353 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:82


[INFO 23-07-13 21:56:37.2165 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpuas7trn4/model/ with prefix a9f82cb7036b485c
[INFO 23-07-13 21:56:37.2210 EDT kernel.cc:1075] Use fast generic engine


i:83


[WARNING 23-07-13 21:56:37.4195 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:37.4195 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:37.4195 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 23-07-13 21:56:37.7099 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp3im99hn6/model/ with prefix a4a43806801e4ae7
[INFO 23-07-13 21:56:37.7136 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:37.8890 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:37.8890 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:37.8890 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boostin

i:84


[INFO 23-07-13 21:56:38.1695 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpn8z3qts0/model/ with prefix fc8da9c395284a27
[INFO 23-07-13 21:56:38.1731 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:38.3469 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:38.3469 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:38.3469 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:85


[INFO 23-07-13 21:56:38.6055 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmph326jim2/model/ with prefix dc7aa8da4690434b
[INFO 23-07-13 21:56:38.6085 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:38.7846 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:38.7846 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:38.7846 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:86


[INFO 23-07-13 21:56:39.1090 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpj__gib0p/model/ with prefix 240551a9f5104295
[INFO 23-07-13 21:56:39.1143 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:39.2907 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:39.2908 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:39.2908 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:87


[INFO 23-07-13 21:56:39.5350 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpg760zt9k/model/ with prefix f62efa1ce03a4f20
[INFO 23-07-13 21:56:39.5378 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:39.7071 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:39.7071 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:39.7071 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:88


[INFO 23-07-13 21:56:39.9294 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpcfyvp5g4/model/ with prefix be8f6585c47844e5
[INFO 23-07-13 21:56:39.9310 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:40.1107 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:40.1107 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:40.1107 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:89


[INFO 23-07-13 21:56:40.3586 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmptookvz95/model/ with prefix a1fb233679e64d10
[INFO 23-07-13 21:56:40.3614 EDT abstract_model.cc:1312] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 23-07-13 21:56:40.3614 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:40.5328 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:40.5328 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:40.5328 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:90


[INFO 23-07-13 21:56:40.7785 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpka0oxj1k/model/ with prefix 9ab8a785dcee4148
[INFO 23-07-13 21:56:40.7823 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:40.9561 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:40.9561 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:40.9561 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:91


[INFO 23-07-13 21:56:41.2416 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp0j_i_lj7/model/ with prefix 55d2d9c3a503491c
[INFO 23-07-13 21:56:41.2457 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:41.4179 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:41.4179 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:41.4180 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:92


[INFO 23-07-13 21:56:41.7950 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmptpvckfa2/model/ with prefix 13f2a11928de4502
[INFO 23-07-13 21:56:41.8019 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:41.9785 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:41.9785 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:41.9785 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:93


[INFO 23-07-13 21:56:42.2009 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp8ib8_sic/model/ with prefix f795ece4b5b84ce6
[INFO 23-07-13 21:56:42.2028 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:42.3825 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:42.3825 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:42.3825 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:94


[INFO 23-07-13 21:56:42.6238 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp8f1h68px/model/ with prefix 1846822e64ea4a4b
[INFO 23-07-13 21:56:42.6263 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:42.7995 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:42.7995 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:42.7995 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:95


[INFO 23-07-13 21:56:43.0471 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmphbkao9u5/model/ with prefix d04becf9f671437c
[INFO 23-07-13 21:56:43.0497 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:43.2276 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:43.2276 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:43.2276 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:96


[INFO 23-07-13 21:56:43.4821 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp1wb50bk6/model/ with prefix c5f33bbaa66048a6
[INFO 23-07-13 21:56:43.4850 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:43.6596 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:43.6596 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:43.6596 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:97


[INFO 23-07-13 21:56:43.8852 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmpngfzielk/model/ with prefix 713ecfae7f6243a9
[INFO 23-07-13 21:56:43.8871 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:44.0659 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:44.0660 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:44.0660 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:98


[INFO 23-07-13 21:56:44.4912 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp89gd0dw_/model/ with prefix c671129bfd3d4e43
[INFO 23-07-13 21:56:44.4991 EDT kernel.cc:1075] Use fast generic engine
[WARNING 23-07-13 21:56:44.6767 EDT gradient_boosted_trees.cc:1818] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:44.6768 EDT gradient_boosted_trees.cc:1829] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 23-07-13 21:56:44.6768 EDT gradient_boosted_trees.cc:1843] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:99


[INFO 23-07-13 21:56:45.1663 EDT kernel.cc:1243] Loading model from path /var/folders/wp/s6hl_fl57tnf8_1_ndx9w4hw0000gn/T/tmp362dau86/model/ with prefix 1256d471deff4d53
[INFO 23-07-13 21:56:45.1765 EDT kernel.cc:1075] Use fast generic engine


Submission exported to https://www.kaggle.com/competitions/titanic/submissions
